holopy/scattering/theory/scatteringtheory.py

```diff
@@ -16,220 +16,62 @@
 # You should have received a copy of the GNU General Public License
 # along with HoloPy.  If not, see <http://www.gnu.org/licenses/>.
 """
-Base class for scattering theories.  Implements python-based
-calc_intensity and calc_holo, based on subclass's calc_field
 .. moduleauthor:: Jerome Fung <jerome.fung@post.harvard.edu>
 .. moduleauthor:: Vinothan N. Manoharan <vnm@seas.harvard.edu>
 .. moduleauthor:: Thomas G. Dimiduk <tdimiduk@physics.harvard.edu>
 .. moduleauthor:: Brian Leahy <bleahy@g.harvard.edu>
 """
-
-from warnings import warn
-
 import numpy as np
-import xarray as xr
```

Comment:
nice that you were able to completely remove xarray!

Reply:
only because `scatteringtheory.py` is now basically empty though :smile:

I do think that this is useful, because the xarray usage is getting restricted to fewer files, which will make optimizations with it easier.